Configured default search plugin.

abaziz opened the pull request:
Although currently commented as there seems to be some issue.
Need to see how search was working in V1

stakater-user commented:
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins-delivery.stakater.com/job/Stakater/job/StakaterCloudDocs/job/PR-1/1/) has Failed!

stakater-user commented:
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build 2](https://jenkins-delivery.stakater.com/job/Stakater/job/StakaterCloudDocs/job/PR-1/2/) has Failed!

stakater-user commented:
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build 4](https://jenkins-delivery.stakater.com/job/Stakater/job/StakaterCloudDocs/job/PR-1/4/) has Failed!

stakater-user commented:
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build 3](https://jenkins-delivery.stakater.com/job/Stakater/job/StakaterCloudDocs/job/PR-1/3/) has Failed!

stakater-user commented:
@ahmedwaleedmalik Yikes! You better fix it before anyone else finds out! [Build 5](https://jenkins-delivery.stakater.com/job/Stakater/job/StakaterCloudDocs/job/PR-1/5/) has Failed!

stakater-user commented:
@ahmedwaleedmalik Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-1-6`

stakater-user commented:
@ahmedwaleedmalik Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-1-7`

abaziz committed:
Configured default search plugin.
Although currently commented as there seems to be some issue.
Need to see how search was working in V1
